manual/src/main/asciidoc/migrations/migrate-1.6-to-2.0.adoc

```diff
@@ -22,58 +22,58 @@ There are two main steps in preparing your migration to Apache Unomi 2.0:
 
 === Updating applications consuming Unomi
 
-Since Apache Unomi is an engine, you've probably built multiple applications consuming its APIs, you might also have built extensions directly running in Unomi. 
+Since Apache Unomi is an engine, you've probably built multiple applications consuming its APIs, you might also have built extensions directly running in Unomi.
 
-As you begin updating applications consuming Apache Unomi, it is generally a good practice to <<Enable debug mode>>. 
+As you begin updating applications consuming Apache Unomi, it is generally a good practice to <<_enabling_debug_mode,enable debug mode>>.
 Doing so will display any errors when processing events (such as JSON Schema validations), and will provide useful indications towards solving issues.
 
 ==== Data Model changes
 
-There has been changes to Unomi Data model, please make sure to review those in the << what_s_new>> section of the user manual.
+There has been changes to Unomi Data model, please make sure to review those in the <<_whats_new_in_apache_unomi_2_0,What's new in Unomi 2>> section of the user manual.
```

Comment:
Did you verify the anchor was working fine ? (I'm asking specifically for the ' and . )

`<<_whats_new_in_apache_unomi_2_0,What's new in Unomi 2>>` linking to
```
=== What's new in Apache Unomi 2.0
```

Reply:
I actually used IDEA's auto-completion for the anchor so yes I think it should be good.